tools/import-paperless/import-paperless.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/bash
```

Comment:
I'm already sorry for this nitpick :-/ would you mind using `/usr/bin/env bash` instead? It seems more portable …

Reply:
Interesting, didn't know about the [discussion and pros and cons](https://unix.stackexchange.com/questions/206350/what-is-the-difference-if-i-start-bash-with-bin-bash-or-usr-bin-env-bash).

Will change it